### ISSUE TITLE
Unmatched result of aggregation fuzzer test of map_union

### ISSUE THREAD

**Issue by kagamiori** [facebookincubator/velox#3449]
### Description

This is a bug found by velox_aggregation_fuzzer_test. Two logically equivalent plans produce different results. More details can be found in the log.

### Error Reproduction

This failure can be reproduced by rebasing on 2b41694f4 and then running `velox/exec/tests/velox_aggregation_fuzzer_test -seed 3575415659`.

### Relevant logs

```bash
I1207 16:41:46.625551 3399741 AggregationFuzzer.cpp:481] ==============================> Started iteration 0 (seed: 3575415659)
I1207 16:41:46.702518 3399741 AggregationFuzzer.cpp:561] Executing query plan: 
-- Project[expressions: (g0:TINYINT, ROW["g0"]), (g1:BIGINT, ROW["g1"]), (g2:VARBINARY, ROW["g2"]), (p3:ARRAY<TINYINT>, array_sort(map_keys(ROW["a0"])))] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, p3:ARRAY<TINYINT>
  -- Aggregation[SINGLE [g0, g1, g2] a0 := map_union(ROW["c0"])] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
    -- Values[1000 rows in 10 vectors] -> c0:MAP<TINYINT,INTERVAL DAY TO SECOND>, g0:TINYINT, g1:BIGINT, g2:VARBINARY
I1207 16:41:46.883831 3399741 AggregationFuzzer.cpp:576] [ROW ROW<g0:TINYINT,g1:BIGINT,g2:VARBINARY,p3:ARRAY<TINYINT>>: 874 elements, no nulls]
I1207 16:41:47.023860 3399741 AggregationFuzzer.cpp:170] Testing plan #0
I1207 16:41:47.023942 3399741 AggregationFuzzer.cpp:561] Executing query plan: 
-- Project[expressions: (g0:TINYINT, ROW["g0"]), (g1:BIGINT, ROW["g1"]), (g2:VARBINARY, ROW["g2"]), (p3:ARRAY<TINYINT>, array_sort(map_keys(ROW["a0"])))] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, p3:ARRAY<TINYINT>
  -- Aggregation[FINAL [g0, g1, g2] a0 := map_union("a0")] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
    -- Aggregation[PARTIAL [g0, g1, g2] a0 := map_union(ROW["c0"])] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
      -- Values[1000 rows in 10 vectors] -> c0:MAP<TINYINT,INTERVAL DAY TO SECOND>, g0:TINYINT, g1:BIGINT, g2:VARBINARY
I1207 16:41:47.271139 3399741 AggregationFuzzer.cpp:576] [ROW ROW<g0:TINYINT,g1:BIGINT,g2:VARBINARY,p3:ARRAY<TINYINT>>: 874 elements, no nulls]
I1207 16:41:47.433883 3399741 AggregationFuzzer.cpp:173] Testing plan #0 with spilling
I1207 16:41:47.433992 3399741 AggregationFuzzer.cpp:561] Executing query plan: 
-- Project[expressions: (g0:TINYINT, ROW["g0"]), (g1:BIGINT, ROW["g1"]), (g2:VARBINARY, ROW["g2"]), (p3:ARRAY<TINYINT>, array_sort(map_keys(ROW["a0"])))] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, p3:ARRAY<TINYINT>
  -- Aggregation[FINAL [g0, g1, g2] a0 := map_union("a0")] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
    -- Aggregation[PARTIAL [g0, g1, g2] a0 := map_union(ROW["c0"])] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
      -- Values[1000 rows in 10 vectors] -> c0:MAP<TINYINT,INTERVAL DAY TO SECOND>, g0:TINYINT, g1:BIGINT, g2:VARBINARY
I1207 16:41:47.727171 3399741 AggregationFuzzer.cpp:576] [ROW ROW<g0:TINYINT,g1:BIGINT,g2:VARBINARY,p3:ARRAY<TINYINT>>: 874 elements, no nulls]
I1207 16:41:47.868125 3399741 AggregationFuzzer.cpp:170] Testing plan #1
I1207 16:41:47.868196 3399741 AggregationFuzzer.cpp:561] Executing query plan: 
-- Project[expressions: (g0:TINYINT, ROW["g0"]), (g1:BIGINT, ROW["g1"]), (g2:VARBINARY, ROW["g2"]), (p3:ARRAY<TINYINT>, array_sort(map_keys(ROW["a0"])))] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, p3:ARRAY<TINYINT>
  -- Aggregation[FINAL [g0, g1, g2] a0 := map_union("a0")] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
    -- Aggregation[INTERMEDIATE [g0, g1, g2] a0 := map_union("a0")] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
      -- Aggregation[PARTIAL [g0, g1, g2] a0 := map_union(ROW["c0"])] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
        -- Values[1000 rows in 10 vectors] -> c0:MAP<TINYINT,INTERVAL DAY TO SECOND>, g0:TINYINT, g1:BIGINT, g2:VARBINARY
I1207 16:41:48.203192 3399741 AggregationFuzzer.cpp:576] [ROW ROW<g0:TINYINT,g1:BIGINT,g2:VARBINARY,p3:ARRAY<TINYINT>>: 874 elements, no nulls]
I1207 16:41:48.344101 3399741 AggregationFuzzer.cpp:173] Testing plan #1 with spilling
I1207 16:41:48.344189 3399741 AggregationFuzzer.cpp:561] Executing query plan: 
-- Project[expressions: (g0:TINYINT, ROW["g0"]), (g1:BIGINT, ROW["g1"]), (g2:VARBINARY, ROW["g2"]), (p3:ARRAY<TINYINT>, array_sort(map_keys(ROW["a0"])))] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, p3:ARRAY<TINYINT>
  -- Aggregation[FINAL [g0, g1, g2] a0 := map_union("a0")] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
    -- Aggregation[INTERMEDIATE [g0, g1, g2] a0 := map_union("a0")] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
      -- Aggregation[PARTIAL [g0, g1, g2] a0 := map_union(ROW["c0"])] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
        -- Values[1000 rows in 10 vectors] -> c0:MAP<TINYINT,INTERVAL DAY TO SECOND>, g0:TINYINT, g1:BIGINT, g2:VARBINARY
I1207 16:41:48.690418 3399741 AggregationFuzzer.cpp:576] [ROW ROW<g0:TINYINT,g1:BIGINT,g2:VARBINARY,p3:ARRAY<TINYINT>>: 874 elements, no nulls]
I1207 16:41:48.839833 3399741 AggregationFuzzer.cpp:170] Testing plan #2
I1207 16:41:48.839898 3399741 AggregationFuzzer.cpp:561] Executing query plan: 
-- Project[expressions: (g0:TINYINT, ROW["g0"]), (g1:BIGINT, ROW["g1"]), (g2:VARBINARY, ROW["g2"]), (p3:ARRAY<TINYINT>, array_sort(map_keys(ROW["a0"])))] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, p3:ARRAY<TINYINT>
  -- Aggregation[FINAL [g0, g1, g2] a0 := map_union("a0")] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
    -- LocalPartition[REPARTITION] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
      -- Aggregation[PARTIAL [g0, g1, g2] a0 := map_union(ROW["c0"])] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
        -- Values[300 rows in 3 vectors] -> c0:MAP<TINYINT,INTERVAL DAY TO SECOND>, g0:TINYINT, g1:BIGINT, g2:VARBINARY
      -- Aggregation[PARTIAL [g0, g1, g2] a0 := map_union(ROW["c0"])] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
        -- Values[300 rows in 3 vectors] -> c0:MAP<TINYINT,INTERVAL DAY TO SECOND>, g0:TINYINT, g1:BIGINT, g2:VARBINARY
      -- Aggregation[PARTIAL [g0, g1, g2] a0 := map_union(ROW["c0"])] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
        -- Values[200 rows in 2 vectors] -> c0:MAP<TINYINT,INTERVAL DAY TO SECOND>, g0:TINYINT, g1:BIGINT, g2:VARBINARY
      -- Aggregation[PARTIAL [g0, g1, g2] a0 := map_union(ROW["c0"])] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
        -- Values[200 rows in 2 vectors] -> c0:MAP<TINYINT,INTERVAL DAY TO SECOND>, g0:TINYINT, g1:BIGINT, g2:VARBINARY
I1207 16:41:49.037118 3399741 AggregationFuzzer.cpp:576] [ROW ROW<g0:TINYINT,g1:BIGINT,g2:VARBINARY,p3:ARRAY<TINYINT>>: 874 elements, no nulls]
I1207 16:41:49.175185 3399741 AggregationFuzzer.cpp:173] Testing plan #2 with spilling
I1207 16:41:49.175271 3399741 AggregationFuzzer.cpp:561] Executing query plan: 
-- Project[expressions: (g0:TINYINT, ROW["g0"]), (g1:BIGINT, ROW["g1"]), (g2:VARBINARY, ROW["g2"]), (p3:ARRAY<TINYINT>, array_sort(map_keys(ROW["a0"])))] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, p3:ARRAY<TINYINT>
  -- Aggregation[FINAL [g0, g1, g2] a0 := map_union("a0")] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
    -- LocalPartition[REPARTITION] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
      -- Aggregation[PARTIAL [g0, g1, g2] a0 := map_union(ROW["c0"])] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
        -- Values[300 rows in 3 vectors] -> c0:MAP<TINYINT,INTERVAL DAY TO SECOND>, g0:TINYINT, g1:BIGINT, g2:VARBINARY
      -- Aggregation[PARTIAL [g0, g1, g2] a0 := map_union(ROW["c0"])] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
        -- Values[300 rows in 3 vectors] -> c0:MAP<TINYINT,INTERVAL DAY TO SECOND>, g0:TINYINT, g1:BIGINT, g2:VARBINARY
      -- Aggregation[PARTIAL [g0, g1, g2] a0 := map_union(ROW["c0"])] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
        -- Values[200 rows in 2 vectors] -> c0:MAP<TINYINT,INTERVAL DAY TO SECOND>, g0:TINYINT, g1:BIGINT, g2:VARBINARY
      -- Aggregation[PARTIAL [g0, g1, g2] a0 := map_union(ROW["c0"])] -> g0:TINYINT, g1:BIGINT, g2:VARBINARY, a0:MAP<TINYINT,INTERVAL DAY TO SECOND>
        -- Values[200 rows in 2 vectors] -> c0:MAP<TINYINT,INTERVAL DAY TO SECOND>, g0:TINYINT, g1:BIGINT, g2:VARBINARY
I1207 16:41:49.467693 3399741 AggregationFuzzer.cpp:576] [ROW ROW<g0:TINYINT,g1:BIGINT,g2:VARBINARY,p3:ARRAY<TINYINT>>: 874 elements, no nulls]
velox/exec/tests/utils/QueryAssertions.cpp:986: Failure
Value of: false
  Actual: false
Expected: true
Expected 874, got 874
2 extra rows, 2 missing rows
2 of extra rows:
	null | null | null | [0,2,3,3,4,6,6,7,13,14,18,20,21,22,27,30,32,33,34,36,39,41,41,42,45,47,49,50,53,53,55,56,57,59,61,63,69,72,76,76,79,81,82,84,85,87,87,88,90,91,92,93,94,97,99,100,101,103,105,106,107,109,112,112,115,117,118,122,124,126,127]
	33 | null | null | [8,17,18,24,36,45,47,47,55,61,71,81,87,89,94,102,113,114,119,122]

2 of missing rows:
	null | null | null | [0,2,3,4,6,7,13,14,18,20,21,22,27,30,32,33,34,36,39,41,42,45,47,49,50,53,55,56,57,59,61,63,69,72,76,79,81,82,84,85,87,88,90,91,92,93,94,97,99,100,101,103,105,106,107,109,112,115,117,118,122,124,126,127]
	33 | null | null | [8,17,18,24,36,45,47,55,61,71,81,87,89,94,102,113,114,119,122]

Unexpected results
E1207 16:41:49.663960 3399741 Exceptions.h:68] Line: velox/exec/tests/AggregationFuzzer.cpp:788, Function:testPlan, Expression: assertEqualResults({expected.result}, {actual.result}) Logically equivalent plans produced different results, Source: RUNTIME, ErrorCode: INVALID_STATE
I1207 16:41:49.670545 3399741 AggregationFuzzer.cpp:423] Persisted input: /home/weihe/fuzzer_repro/velox_vector_V9OA4e and plan: /home/weihe/fuzzer_repro/velox_plan_Oq7BOS
terminate called after throwing an instance of 'facebook::velox::VeloxRuntimeError'
  what():  Exception: VeloxRuntimeError
Error Source: RUNTIME
Error Code: INVALID_STATE
Reason: Logically equivalent plans produced different results
Retriable: False
Expression: assertEqualResults({expected.result}, {actual.result})
Function: testPlan
File: velox/exec/tests/AggregationFuzzer.cpp
Line: 788
Stack trace:
# 0  std::shared_ptr<facebook::velox::VeloxException::State const> facebook::velox::VeloxException::State::make<facebook::velox::VeloxException::make(char const*, unsigned long, char const*, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, bool, facebook::velox::VeloxException::Type, std::basic_string_view<char, std::char_traits<char> >)::$_0>(facebook::velox::VeloxException::Type, facebook::velox::VeloxException::make(char const*, unsigned long, char const*, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, bool, facebook::velox::VeloxException::Type, std::basic_string_view<char, std::char_traits<char> >)::$_0)
# 1  facebook::velox::VeloxException::VeloxException(char const*, unsigned long, char const*, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, bool, facebook::velox::VeloxException::Type, std::basic_string_view<char, std::char_traits<char> >)
# 2  facebook::velox::VeloxRuntimeError::VeloxRuntimeError(char const*, unsigned long, char const*, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, std::basic_string_view<char, std::char_traits<char> >, bool, std::basic_string_view<char, std::char_traits<char> >)
# 3  void facebook::velox::detail::veloxCheckFail<facebook::velox::VeloxRuntimeError, char const*>(facebook::velox::detail::VeloxCheckFailArgs const&, char const*)
# 4  facebook::velox::exec::test::(anonymous namespace)::AggregationFuzzer::testPlan(std::shared_ptr<facebook::velox::core::PlanNode const> const&, bool, bool, facebook::velox::exec::test::(anonymous namespace)::ResultOrError const&)
# 5  facebook::velox::exec::test::(anonymous namespace)::AggregationFuzzer::testPlans(std::vector<std::shared_ptr<facebook::velox::core::PlanNode const>, std::allocator<std::shared_ptr<facebook::velox::core::PlanNode const> > > const&, bool, facebook::velox::exec::test::(anonymous namespace)::ResultOrError const&)
# 6  facebook::velox::exec::test::(anonymous namespace)::AggregationFuzzer::verify(std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::vector<std::shared_ptr<facebook::velox::RowVector>, std::allocator<std::shared_ptr<facebook::velox::RowVector> > > const&, bool, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&)
# 7  facebook::velox::exec::test::(anonymous namespace)::AggregationFuzzer::go()
# 8  facebook::velox::exec::test::aggregateFuzzer(std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::vector<std::shared_ptr<facebook::velox::exec::AggregateFunctionSignature>, std::allocator<std::shared_ptr<facebook::velox::exec::AggregateFunctionSignature> > >, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::vector<std::shared_ptr<facebook::velox::exec::AggregateFunctionSignature>, std::allocator<std::shared_ptr<facebook::velox::exec::AggregateFunctionSignature> > > > > >, unsigned long, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > > const&)
# 9  AggregationFuzzerRunner::run(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned long, std::unordered_set<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > > const&)
# 10 main
# 11 __libc_start_call_main
# 12 __libc_start_main_alias_2
# 13 _start

*** Aborted at 1670460109 (Unix time, try 'date -d @1670460109') ***
*** Signal 6 (SIGABRT) (0x37b3c0033e03d) received by PID 3399741 (pthread TID 0x7f86d09a8080) (linux TID 3399741) (maybe from PID 3399741, UID 228156) (code: -6), stack trace: ***
    @ 0000000000010fae folly::symbolizer::(anonymous namespace)::innerSignalHandler(int, siginfo_t*, void*)
                       ./folly/experimental/symbolizer/SignalHandler.cpp:449
    @ 000000000000f6d1 folly::symbolizer::(anonymous namespace)::signalHandler(int, siginfo_t*, void*)
                       ./folly/experimental/symbolizer/SignalHandler.cpp:470
    @ 0000000000000000 (unknown)
    @ 000000000009c9d3 __GI___pthread_kill
    @ 00000000000444ec __GI_raise
    @ 000000000002c432 __GI_abort
    @ 00000000000a3fd4 __gnu_cxx::__verbose_terminate_handler()
    @ 00000000000a1b39 __cxxabiv1::__terminate(void (*)())
    @ 00000000000a1ba4 std::terminate()
    @ 00000000000a1ec1 __cxa_rethrow
    @ 000000000007445f facebook::velox::exec::test::(anonymous namespace)::AggregationFuzzer::verify(std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::vector<std::shared_ptr<facebook::velox::RowVector>, std::allocator<std::shared_ptr<facebook::velox::RowVector> > > const&, bool, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&)
                       ./velox/exec/tests/AggregationFuzzer.cpp:871
    @ 0000000000047e4a facebook::velox::exec::test::(anonymous namespace)::AggregationFuzzer::go()
                       ./velox/exec/tests/AggregationFuzzer.cpp:539
    @ 0000000000044084 facebook::velox::exec::test::aggregateFuzzer(std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::vector<std::shared_ptr<facebook::velox::exec::AggregateFunctionSignature>, std::allocator<std::shared_ptr<facebook::velox::exec::AggregateFunctionSignature> > >, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::vector<std::shared_ptr<facebook::velox::exec::AggregateFunctionSignature>, std::allocator<std::shared_ptr<facebook::velox::exec::AggregateFunctionSignature> > > > > >, unsigned long, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > > const&)
                       ./velox/exec/tests/AggregationFuzzer.cpp:209
    @ 00000000002c8336 AggregationFuzzerRunner::run(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned long, std::unordered_set<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&, std::unordered_map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::hash<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::equal_to<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > > const&)
                       ./velox/exec/tests/AggregationFuzzerRunner.h:93
                       -> ./velox/exec/tests/AggregationFuzzerTest.cpp
    @ 00000000002c66cb main
                       ./velox/exec/tests/AggregationFuzzerTest.cpp:70
    @ 000000000002c656 __libc_start_call_main
    @ 000000000002c717 __libc_start_main_alias_2
    @ 00000000002bc1c0 _start
                       /home/engshare/third-party2/glibc/2.34/src/glibc-2.34/csu/../sysdeps/x86_64/start.S:116
Aborted (core dumped)
```

**Comment by mbasmanova** [comment]
Probably a duplicate of #3402

**Comment by mbasmanova** [comment]
Confirmed that this is fixed in #3291
